### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by dalen**
## What did you implement:

Add a typescript definition for the lib export.
This makes it easier to use this plugin with a webpack.config.ts
like described at https://webpack.js.org/configuration/configuration-languages/#typescript

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+declare module 'serverless-webpack' {
+  const lib: {
```
Comment (HyperBrain) [inline index.d.ts]: Maybe we should list all public members of the lib object here. There are also options and the serverless object. Please see the README for details.

Comment (dalen) [reply index.d.ts]: Good point. I've updated it to add the serverless export and the options now.

**Comment by dalen** [comment]
@HyperBrain I've fixed the requested changes, could you take another look at this?

**Comment by HyperBrain** [comment]
will do later this week

**Pull request merged**